setup.cfg

```diff
@@ -22,6 +22,7 @@ provides =
 [options]
 python_requires = >=3.6.1
 install_requires =
+    bokeh
```

Comment:
I like the html plots A LOT, but I would not make them "mandatory", rather suggested.
I might not _always_ want an html report, for instance if I batch a bunch of files in HCPs. 
In that case we can add bokeh as an extra dependency. What do you @physiopy/all think?

Reply:
If it's an optional thing, we shouldn't enforce the installation. I've added it to the optional installation within the `docs` settings. I thought that made the most sense.

Reply:
I agree with making the html report optional.